LICENSE

```diff
@@ -1,4 +1,4 @@
-NOTICE: Copyright (c) 2017 by the Music Encoding Initiative (MEI) Board (formerly known as "MEI Council").
+NOTICE: Copyright (c) 2017-2023 by the Music Encoding Initiative (MEI) Board (formerly known as "MEI Council").
```

Comment:
Please use typographically better en dash instead of a hyphen.

Reply:
I would like to keep it as plain ASCII text.

Reply:
plain ascii is good for the general license file